dashboard/api.py

```diff
@@ -0,0 +1,227 @@
+"""
+Apis for the dashboard
+"""
+import datetime
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class CourseStatus(object):
+    """
+    Possible statuses for a course for a user
+    """
+    PASSED = 'passed'
+    NOT_PASSED = 'not-passed'
+    CURRENT_GRADE = 'verified-not-completed'
+    UPGRADE = 'enrolled-not-verified'
+    NOT_OFFERED = 'not-offered'
+    OFFERED = 'offered-not-enrolled'
+
+    @classmethod
+    def all_statuses(cls):
```

Comment:
It looks like we aren't using this function anywhere but the tests. Do we need it in the future?